### PR TITLE
feat(register):updates screen integration

### PR DESCRIPTION
**Pull request by poojadsit**
feat(register):updates screen integration

### CODE REVIEW

File: DVSRegister/Controllers/RegisterController.cs
Comment (AiswaryaBEIS) [inline DVSRegister/Controllers/RegisterController.cs:1]: Task<ActionResult<List<RegisterPublishLogDto>>> Task<IActionResult>  will be enough , as we are returning view

File: DVSRegister/Models/Register/RegisterPublishLogsViewModel.cs
Comment (AiswaryaBEIS) [inline DVSRegister/Models/Register/RegisterPublishLogsViewModel.cs:1]: public List<ProviderDto>? Providers { get; set; }   if not using in view, can be removed

File: DVSRegister/Views/Register/Updates.cshtml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿@using DVSRegister.CommonUtility.Models.Enums;
+@model RegisterPublishLogsViewModel
+@{
+    ViewData["Title"] = "Updates";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper govuk-main-wrapper--l" id="main-content" role="main">
```
[the remark this answers is not in the document]
Comment (poojadsit) [reply DVSRegister/Views/Register/Updates.cshtml:9]: Should I use same layout (classes and structure) for this page?

File: DVSRegister/Models/Register/RegisterPublishLogsViewModel.cs
Comment (poojadsit) [reply DVSRegister/Models/Register/RegisterPublishLogsViewModel.cs:1]: I thought pages before this one might need ProviderId, if not I'll remove it.

Comment (AiswaryaBEIS) [reply DVSRegister/Models/Register/RegisterPublishLogsViewModel.cs:1]: We dont  need provider id as of now , as it is fetching full list

**Review by AiswaryaBEIS — APPROVED** [review]
Looks good

**Pull request merged**